doc/cephfs/fs-volumes.rst

```diff
@@ -365,6 +365,14 @@ This command resizes the subvolume quota, using the size specified by
 ``new_size``.  The ``--no_shrink`` flag prevents the subvolume from shrinking
 below the current "used size" of the subvolume.
 
+Resizing can also be done using human-friendly units::
+
+  ceph fs subvolume resize foo subvol1 100KiB
+  ceph fs subvolume resize foo subvol1 100.45KiB
```

Comment:
Suggestion: We should also add one example of resize in SI unit

Reply:
https://github.com/ceph/ceph/compare/705f4f4ef09a2b8e82fb0bcfa4868c6db114ab98..94bdd112f94f6f80f0e86f1aad8807f83576dc41